spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -165,12 +165,12 @@
     "viewersPersonalExperience": "personal",
     "viewersPersonalResponse": "response",
     "viewersReference": "reference;Viewer's;menje",
-    "viewersContributionNote": "Viewer's contribution note",
+    "viewersContributionNote": "Viewer's contribution note\nNext line",
     "referenceLocal": "makasi",
     "referenceWorldcat": "Chelse",
-    "referenceNote": "risasi;bad man",
+    "referenceNote": "bad man;risasi",
```

Comment:
Related to the Mapper processing multi-authority fields in a certain order. Changed order of reference fields in XML, so needed to change the order of the referenceNote values accordingly